routing/record/validation.go

```diff
@@ -100,3 +112,18 @@ func CheckRecordSig(r *pb.Record, pk ci.PubKey) error {
 	}
 	return nil
 }
+
+type RecordSorter []*pb.Record
+
+func (rs RecordSorter) Less(i, j int) bool {
+	mv, _ := MoreValid(rs[j], rs[i])
```

Comment:
I really think its okay to have different validity types, maybe for a period of time i'm planning on updating frequently so i set it to have a short TTL, then later I want to do something more permanent like "This record is valid until the bitcoin block XYZ". I think its a totally valid use case